README.md

```diff
@@ -1,5 +1,7 @@
 # bricsauthenticator
 
+[![codecov](https://codecov.io/gh/owhere/bricsauthenticator/branch/tests/graph/badge.svg?token=3IGKBYAIS1)](https://codecov.io/gh/owhere/bricsauthenticator)
```

Comment:
I think this badge might be showing the coverage for `owhere/bricsauthenticator`, not `isambard-sc/bricsauthenticator`.

It looks like the upload to Codecov is failing in Actions, see e.g. <https://github.com/isambard-sc/bricsauthenticator/actions/runs/13262845648/job/37023173498>

```text
 [2025-02-11T12:25:43.623Z] ['error'] There was an error running the uploader: Error uploading to https://codecov.io: Error: There was an error fetching the storage URL during POST: 429 - {"message":"Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 1914s."}

[2025-02-11T12:25:43.623Z] ['info'] Codecov will exit with status code 0. If you are expecting a non-zero exit code, please pass in the `-Z` flag
```

Reply:
I've made an issue to track this: https://github.com/isambard-sc/bricsauthenticator/issues/10